logict-sequence.cabal

```diff
@@ -51,7 +53,8 @@ library
     build-depends: mtl >=2.0 && <2.3
     build-depends: type-aligned >= 0.9.6 && < 0.10
     build-depends: sequence >= 0.9.8 && < 0.10
-    build-depends: logict
+    build-depends: logict >= 0.7.1.0
```

Comment:
I wonder if we should also have an upper bound like maybe `< 0.8`, or didn't cabal get some fancy new operator for bounds like `==^*&&@@<>` (I really can't remember what it was, sorry).

Reply:
You're asking me? That's semi-hilarious. That does seem to be popular, though.